### DISCUSSION TITLE
Initial platform targets?

### DISCUSSION THREAD

**Discussion by jacksongoode** [dzirbel/kotify#3] [General]
Been following along, and wondering which platforms you will target first for a release?

**Comment by dzirbel** [comment]
Thanks for the question! I am planning to create releases for Windows (.msi and .exe), macOS (.dmg and .pkg), and Linux (.deb and .rpm), as well as a .jar file that can be run on any platform, but I can't make any promises yet since I haven't set up the release pipelines.
